addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -547,6 +551,22 @@ return AbstractRenderer.extend({
         options.plugins.push(createYearCalendarView(FullCalendar, options));
         return options;
     },
+    /**
+     * Compute the string to find the good event.
+     * @param {jQueryElement} info
+     * @returns {String}
+     */
+    _findActualEvent: function (info) {
```

Comment:
maybe rename to "_computeEventSelector" ? or something like that.
as you don't return the actualEvent but the selector to find it.